### PR TITLE
Comment involvement

### PR DESCRIPTION
**Pull request by DhuruvSingh**
# **Display Comments & Add new comment form**

- ✔️**When the popup loads, the web app retrieves data from the [Involvement API](https://www.notion.so/microverse/Involvement-API-869e60b5ad104603aa6db59e08150270) to show the item comments.**

- ✔️**Implemented the "add new comment" form on the Comments pop-up according to the wireframe:**

> **Kindly review and give your valuable feedback**

### CODE REVIEW

File: src/modules/showPopup.js [hunk 1]
```diff
@@ -50,6 +68,76 @@ const showPopup = async () => {
...
+      const idShow = 0;
+
+      submitBtn.addEventListener('click', () => {
+        // eslint-disable-next-line camelcase
+        const item_id = idShow;
+        const usernameInput = document.getElementById('username');
+        const commentInput = document.getElementById('comment');
+        const username = usernameInput.value;
+        const comment = commentInput.value;
+        const dataToSend = JSON.stringify({ item_id, username, comment });
```
Comment (Peter1907) [inline src/modules/showPopup.js:81]: Here, you are using the same id for all of the items
A way this is to use the id of the comment buttons to send data to the API

**Review by Peter1907 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi @DhuruvSingh , 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
Git flow is followed. :heavy_check_mark:
The comments section is hitting the API correctly.:heavy_check_mark:

## To Improve 🙏
- [ ] Each comment pop-up should send & receive unique data from the API. however, they all have the same data (comments) now ... for more details please see the comment below:point_down:

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

[Optional] you can display a new comment instantly in the page once it has been sent to the api rather than waiting to hit the API after reloading the pop-up.

## Happy coding!👏👏👏

#### Feel free to leave any questions or comments in the PR thread or on our WhatsApp group if something is not 100% clear.

**Review by Peter1907 — APPROVED** [review]
# Status: APPROVED 🟢 

Hi @DhuruvSingh  ✋

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! :tada: :clap: :clap: :100:
you-did-it-celebration

## Highlights!
- Followed Gitflow ✔️ 
- Good commit messages ✔️
- Linters are passing ✔️
- Descriptive Pull request ✔️

## Optional suggestions

- [ ] Kindly implement the comment below then merge the branch

##### _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

##### Cheers and Happy coding!:clap::clap::clap:
##### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

File: src/modules/showPopup.js [hunk 2]
```diff
@@ -120,21 +119,26 @@ const showPopup = async () => {
...
-              )
-              .join(' ');
-            commentsDisplay.innerHTML = gege;
+                )
+                .join(' ');
+              commentsDisplay.innerHTML = gege;
+            // eslint-disable-next-line no-empty
+            } else {
+
+            }
```
[the remark this answers is not in the document]
Comment (DhuruvSingh) [reply src/modules/showPopup.js]: done

**Pull request merged**